source/jucePlugin/PluginEditor.cpp

```diff
@@ -14,6 +14,9 @@ AudioPluginAudioProcessorEditor::AudioPluginAudioProcessorEditor(AudioPluginAudi
     // editor's size to whatever you need it to be.
 	setSize(800, 400);
 
+	// Resizable UI
```

Comment:
I know this is temporary. But if already adding this. IMHO worth adding resize limits.

Reply:
agree..

setResizeLimits(800,400,800,1600);  

sound reasonable to you?

Reply:
(ill add spaces after the comma's of course, lol)

Reply:
Feel free to make even 4k Max size

Reply:
i think 1600 is plenty...  its a temporary hack..  more the 4x the parameters are now available at one time without scrolling..   just a quick and easy quality of life improvement for the beta testers...  done!